### ISSUE TITLE
The number of debug messages is too damn high!

### ISSUE THREAD

**Issue by martonr** [AnimeniacYuuto/EnhancedInventories#51]
In **gui.slot.SlotCraftingExtended.getOresFromRecipe()** the **LogHelperEI.Debug()** method is called, and the message is recorded in the logs always since in the *Debug()* method does not use the *debug* field value to determine the output.

```scala
def Debug(obj:Any) {
    if(debug)
        Info(obj);
    LogSpecial(Level.DEBUG, obj); //This always executes...
}
```

When using the Auto Assembler in an automated manner the logs get filled up with the ("Checking ores at #") debug messages making them several hundred megabytes in size in minutes.

**Comment by canilsen** [comment]
It's the same when using the hopper upgrade I think. Constant spam

**Comment by Pontiac76** [comment]
I was prepared to blame another mod as the server console is reporting stuff periodically, but I started a trace against the actual offending massive log file, and found that Enhanced Inventories is the primary culprit.

mc@core regrowth/logs$ cat fml-server-latest.log | grep 06:15:37 | wc                                                     1196   10728   92200      

That means every second, around 1k lines are being added to the log file.

This is pretty critical concerning drive space.

**Comment by canilsen** [comment]
1k line every second, wtf

**Comment by AnimeniacYuuto** [comment]
fixed as of b38d62de1a54b8238afabe302fe0a65fa1777c07 appologize
